jupyterhub/Dockerfile

```diff
@@ -55,8 +56,10 @@ ENV SIRF_PATH "/opt/SIRF-SuperBuild/sources/SIRF"
 #Suppress output from Gadgetron which gives some problems on notebooks (QUIERO)
 ENV GADGETRON_LOG_MASK ""
 RUN echo $PATH
+USER root
+RUN echo "export OMP_NUM_THREADS=\$(python -c 'import multiprocessing as mc; print(mc.cpu_count() // 2)')" > /usr/local/bin/before-notebook.d/omp_num_threads.sh
```

Comment:
this works because `/usr/local/bin/before-notebook.d/*.sh` is auto-sourced by `/usr/local/bin/start.sh`

Reply:
and it will be run every time the user starts and it will depend on the machine that runs the code